src/scr.c

```diff
@@ -711,40 +730,79 @@ static int scr_get_params()
   /* set logging */
   if ((value = scr_param_get("SCR_LOG_ENABLE")) != NULL) {
     scr_log_enable = atoi(value);
+    if (scr_my_rank_world == 0) {
+      // 5. Dump SCR parameters - SCR_LOG_ENABLE
+      scr_dbg(scr_debug, "SCR_LOG_ENABLE is enabled.");
+    }
   }
 
   /* check whether SCR logging DB is enabled */
   if ((value = scr_param_get("SCR_LOG_TXT_ENABLE")) != NULL) {
     scr_log_txt_enable = atoi(value);
+    if (scr_my_rank_world == 0) {
+      // 6. Dump SCR parameters - SCR_LOG_TXT_ENABLE
+      scr_dbg(scr_debug, "SCR logging DB is enabled, SCR_LOG_TXT_ENABLE = %d", scr_log_txt_enable);
```

Comment:
Oh, the first argument of scr_dbg specifies the debug level at which the message should be printed.  Let's use level=1 for these, meaning they will print when someone sets ``SCR_DEBUG=1``.
```suggestion
      scr_dbg(1, "SCR logging DB is enabled, SCR_LOG_TXT_ENABLE = %d", scr_log_txt_enable);
```